api/predict.py

```diff
@@ -54,9 +60,12 @@ def post(self):
         try:
             preds = self.model_wrapper.predict(input_json)
         except:  # noqa
-            abort(400, "Invalid input to Pinyin converter, please check!")
+            abort(400, "The input word(s) is/are not valid utf-8 encoded Chinese word(s)")
```

Comment:
Something like this might be less wordy:
```suggestion
            abort(400, "The input does not contain valid utf-8 encoded Chinese characters")
```